tools/snyk-update.py

```diff
@@ -0,0 +1,24 @@
+
+import json
+import os
+
+# Load results
+scan_file = open('scan.json')
+scan = json.load(scan_file)
+scan_file.close()
+
+# Make changes to fixable results
+remediations = scan['remediation']['pin']
+for k, v in remediations.items():
+    package, old_version = k.split('@')
+    new_version = v['upgradeTo'].split('@')[1]
+    print(package, old_version, new_version)
+
+    # Remove old version
+    os.system('sed -i "/%s/d" ckan/requirements.in' % (package + "==" + old_version))
+    os.system('sed -i "/%s/d" ckan/requirements.in' % (package + ">=" + old_version))
+
+    # Add new version
+    os.system("echo '%s' >> ckan/requirements.in" % (package + ">=" + new_version))
```

Comment:
do we want snyk PR's to be fuzzy versioned or fixed?

Reply:
The "loose" `requirements.in` will be fuzzy, but the "definitive" `requirements.txt` will be fixed.